exporters/otlp/src/otlp_grpc_log_exporter_factory.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef ENABLE_LOGS_PREVIEW
+
+// MUST be first (absl)
+#  include "opentelemetry/exporters/otlp/otlp_grpc_log_exporter.h"
+
+#  include "opentelemetry/exporters/otlp/otlp_grpc_exporter_options.h"
+#  include "opentelemetry/exporters/otlp/otlp_grpc_log_exporter_factory.h"
+#  include "opentelemetry/sdk/common/global_log_handler.h"
```

Comment:
Do we need global_log_handler.h header here?

Reply:
Thanks.

No we don't, removed.